### PR TITLE
Surface `karma' via GraphQL API

### PR DESCRIPTION
**Pull request by jcorrado**
ref: #70

### CODE REVIEW

File: src/yetibot/core/webapp/resolvers.clj [hunk 1]
```diff
@@ -92,7 +92,8 @@
       {:username username
        :is_active active?
        :id id
-       :last_active last-active})))
+       :last_active last-active
+       :karma (karma/get-score (str "@" id))})))
```
Comment (jcorrado) [inline src/yetibot/core/webapp/resolvers.clj]: :nauseated_face:

Comment (devth) [reply src/yetibot/core/webapp/resolvers.clj]: Temporary hack :)

**Comment by devth** [comment]
@jcorrado lgtm - is this ready or is there remaining work?

**Comment by jcorrado** [comment]
@devth I was planning to add some of the karma reporting aggregates we discussed, before calling this PR good to go.  That being said, if you wanted to merge this tiny bit in to get something up in GQL, that's fine with me - I can cut another branch to work on the aggregates.  Your call, please.

**Comment by devth** [comment]
I'm fine waiting. Just wanted to make sure you weren't blocked on me reviewing/merging.

**Comment by jcorrado** [comment]
Back to you for review, @devth.

File: src/yetibot/core/webapp/resolvers.clj [hunk 2]
```diff
@@ -105,3 +107,10 @@
 (defn crons-resolver
   [context {:keys [] :as args} value]
   (db.cron/query {:query/identifiers identity}))
+
+(defn karmas-resolver
+  [context {:keys [] :as args} value]
+  (map (fn [{:keys [:user-id :score]}]
```
Comment (devth) [inline src/yetibot/core/webapp/resolvers.clj]: Minor: it's idiomatic to eschew the `:` on the named keys inside the vector e.g. `{:keys [user-id score]}`

Comment (jcorrado) [reply src/yetibot/core/webapp/resolvers.clj]: Oh yeah!  Thanks for catching that :raised_hands: .

**Comment by jcorrado** [comment]
For future reference:
```
~/projects/yetibot/yetibot.core karma-in-graphql-api
❯ curl -s 'http://localhost:3003/graphql' \
     -H 'Accept: application/json' \
     --data 'query={karmas{user_id, score}}' \
     --compressed | jq .
{
  "data": {
    "karmas": [
      {
        "user_id": "@U3CTLM3DX",
        "score": 2
      },
      {
        "user_id": "jung",
        "score": 1
      }
    ]
  }
}
```

**Pull request merged**